### ISSUE TITLE
ncurses for build

### ISSUE THREAD

**Issue by tofutim** [tofutim/raspbian-crosstool-ng-toolchain#1]
```
[INFO ]  Installing ncurses for build
[EXTRA]    Configuring ncurses
[EXTRA]    Building ncurses
[ERROR]    gmake[2]: *** [Makefile:219: fallback.c] Error 1
[ERROR]    gmake[2]: *** Waiting for unfinished jobs....
[ERROR]    gmake[1]: *** [Makefile:114: all] Error 2
```

**Comment by tofutim** [comment]
https://github.com/crosstool-ng/crosstool-ng/issues/696

**Comment by tofutim** [comment]
fixed by switching to ncurses 6.1 in 51a711ce354ae2dc5ce5200187430864ba2512a7
